lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
Or, simply not define default values for optional parameters (which doesn't make sense anyway). In which case: I need an implicit presence validator for all default settings; For all other validators,if a parameter isn't among default settings, it should be treated as optional.

This would require documentation, but I think the logic is pretty clear/sensible?